ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/agent/MLPlanExecuteAndReflectAgentRunner.java

```diff
@@ -109,18 +113,21 @@ public class MLPlanExecuteAndReflectAgentRunner implements MLAgentRunner {
     public static final String SYSTEM_PROMPT_FIELD = "system_prompt";
     public static final String QUESTION_FIELD = "question";
     public static final String MEMORY_ID_FIELD = "memory_id";
+    public static final String PARENT_INTERACTION_ID_FIELD = "parent_interaction_id";
     public static final String TENANT_ID_FIELD = "tenant_id";
     public static final String RESULT_FIELD = "result";
     public static final String RESPONSE_FIELD = "response";
     public static final String STEP_RESULT_FIELD = "step_result";
-    public static final String REACT_AGENT_ID_FIELD = "reAct_agent_id";
-    public static final String REACT_AGENT_MEMORY_ID_FIELD = "reAct_agent_memory_id";
+    public static final String REACT_AGENT_ID_FIELD = "react_agent_id";
+    public static final String REACT_AGENT_MEMORY_ID_FIELD = "react_agent_memory_id";
```

Comment:
ReAct Agent is the implementation details. I think it could be other agent in future. Is it better to use `executor_agent_memory_id` ?